src/frontend/src/btc/utils/btc-transactions.utils.ts

```diff
@@ -63,16 +60,16 @@ export const mapBtcTransaction = ({
 
 	const utxosFee = totalInputValue - totalOutputValue;
 
+	// +1 is needed to account for the block where the transaction was first included
 	const confirmations = nonNullish(block_index)
-		? latestBitcoinBlockHeight - block_index
+		? latestBitcoinBlockHeight - block_index + 1
```

Comment:
I updated the existing tests and mocks, `confirmations` value is now either a number that >= 1 or `undefined`. Please let me know in case you have some additional test cases in mind.